### PR TITLE
Add history

### PR DESCRIPTION
**Pull request by mschader**
This PR is in preparation and to track status of the implementation of a history model as stated in #13

### CODE REVIEW

**Comment by fleaz** [comment]
Quick thought after looking at the model: Don't you also need the corresponding group as a foreign key? Otherwise you have a event with "new user joined, userid: 4" but don't know which group this belongs to :D

**Comment by mschader** [comment]
The event_id is bound to the payment or transaction id, so you can get all of these for a group and then query the ids from the history entries.

Another aspect is the deletion of entries should not delete them completly from the database because then it would be hard to get them for the history after deletion. Will look into that.

**Comment by fleaz** [comment]
This fixes #13

File: opensplit/models.py [hunk 1]
```diff
@@ -121,3 +121,34 @@ def jsonify(self):
...
+
+
+class Event(db.Model):
+    __tablename__ = 'history'
+    id = Column(Integer, primary_key=True)
+    date = Column(DateTime, nullable=False, default=datetime.utcnow)
+    group_id = Column(Integer, ForeignKey('group.id'), nullable=False)
+    event_type = Column(String(60), nullable=False)
+    user_id = Column(Integer, ForeignKey('user.id'))
+    expense_id = Column(Integer, ForeignKey('user.id'))
```
Comment (mweinelt) [inline opensplit/models.py:133]: Foreign-key copy-pasta issue?

**Pull request merged**